pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -34,7 +34,7 @@ func NewResourceSyncController(
 	}
 	if err := resourceSyncController.SyncConfigMap(
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespaceName, Name: "client-ca"},
-		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.KubeAPIServerNamespaceName, Name: "client-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.MachineSpecifiedGlobalConfigNamespace, Name: "kube-apiserver-client-ca"},
```

Comment:
isn't that even called managed-kube-apiserver-client-ca-bundle now?

Reply:
@sttts i used the name I saw in the events, but it is possible... @deads2k ?

Reply:
no, it's not.

The final (assembled) client-ca is written to `openshift-kube-apiserver/client-ca` here: https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L266